src/components/Cart/Cart.js

```diff
@@ -37,12 +37,20 @@ const Cart = ({
             </button>
           </div>
 
+          {/* EMPTY CART */}
+
           {cartItems.length === 0 ? (
             <p className={cartStyles.emptyCart}>The cart is empty.</p>
           ) : null}
 
+          {/* CART ITEMS */}
+
           {cartItems.map((item, index) => (
-            <div key={index} className={cartStyles.wrapperInner}>
+            <Link
```

Comment:
@sandrina-p could you check out the cart link paths? For some reason, it only works on the first link click to return e.g. http://localhost:3001/product/product1 but each following click on cart links it adds another /product/...

Reply:
The link is wrong. When it's an absolute path relative to the domain root, you need `/` at the beginning. Otherwise, it will append to the current path.

```diff
- to={`product/${item.path}`}
+ to={`/product/${item.path}`}
```

Reply:
Ah I completely missed that. Thanks Sandrina, you're a lifesaver!